### PR TITLE
Initial helix docs

### PR DESCRIPTION
**Pull request by HaoK**
@natemcmaster @ryanbrandenburg review initial helix doc?

### CODE REVIEW

File: docs/Helix.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+==============================
+
+Helix is the distributed test platform that we use to run tests.  We build a helix payload that contains the publish directory of every test project that we want to test
+send a job with with this payload to a set of queues for the various combinations of OS that we want to test 
+for example: `Windows.10.Amd64.ClientRS4.VS2017.Open`, `OSX.1012.Amd64.Open`, `Ubuntu.1810.Amd64.Open`. Helix takes care of unzipping, running the job, and reporting results.
+
+For more info about helix see: [SDK](https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Helix/Sdk/Readme.md), [JobSender](https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Helix/Sdk/Readme.md) 
+
+## How do I look at the results of a helix run?
+There's a link embedded in the build.cmd log of the helix target on AzoDO, near the bottom right that will look something like this:
```
Comment (natemcmaster) [inline docs/Helix.md]:
```suggestion
There's a link embedded in the build.cmd log of the helix target on Azure Pipelines, near the bottom right that will look something like this:
```

File: docs/Helix.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+The link will take you to an overview of all the tests with clickable links to the logs and each run broken down by queue. 
+
+All of the helix runs for aspnetcore can be found here https://mc.dot.net/#/user/aspnetcore/builds
+
+## What do I do if a test fails?
+You can simulate how most tests locally:
+```
+dotnet publish
+cd <the publish directory>
+dotnet vstest
```
Comment (natemcmaster) [inline docs/Helix.md]:
```suggestion
dotnet vstest My.Tests.dll
```

File: docs/Helix.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Running helix tests locally
+`.\build.cmd /t:Helix /p:IsHelixJob=true`
+
+This will restore, and then publish all of the test projects including some bootstrapping scripts that will install the correct dotnet runtime/sdk before running the test assemblies on the helix machine, and upload the job to helix, it won't wait for the jobs to complete, but you can go to https://mc.dot.net/#/user/aspnetcore/builds and look for a source that matches private-yourusername
+
+## How to skip tests on helix
+There are two main ways to opt out of helix
+- Skipping the entire test project via `<BuildHelixPayload>false</BuildHelixPayload>` in csproj (the default value for this is IsTestProject). This will
+- Skipping an individual test via '[SkipOnHelix]` which might require including a compile reference to: `<Compile Include="..\..\..\..\test\xunit\SkipOnHelixAttribute.cs" />`
```
Comment (natemcmaster) [inline docs/Helix.md]:
```suggestion
- Skipping an individual test via `[SkipOnHelix]` which might require including a compile reference to: `<Compile Include="$(SharedSourceRoot)test\xunit\SkipOnHelixAttribute.cs" />`
```

**Review by natemcmaster — APPROVED** [review]
A few suggestions. Otherwise LGTM.

File: docs/Helix.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+2019-02-07T22:01:26.0028154Z   Job d597c581-f81b-446c-8daf-7c6511b526f7 is completed with 108 finished work items.
+2019-02-07T22:06:33.6898567Z   Job 82f27d4c-9099-4f0e-b383-870c24d8dc2c is completed with 108 finished work items.
+```
+
+The link will take you to an overview of all the tests with clickable links to the logs and each run broken down by queue. 
+
+All of the helix runs for aspnetcore can be found here https://mc.dot.net/#/user/aspnetcore/builds
+
+## What do I do if a test fails?
+You can simulate how most tests locally:
```
Comment (Eilon) [inline docs/Helix.md]: `how most tests *run*`?

**Review by Eilon — APPROVED** [review]
Looks alright to me.

**Pull request merged**